internal/use_cases/create_pull_request.go

```diff
@@ -43,8 +48,50 @@ type CreatePullRequest struct {
 	BranchProvider          domain.BranchProvider
 }
 
+func (cpr *CreatePullRequest) normalizeTemplatePath(path string) (string, error) {
+	if path == "" {
+		return "", nil
+	}
+
+	if !filepath.IsAbs(path) {
+		repoRoot, err := cpr.Git.GetRepositoryRoot()
+		if err != nil {
+			return "", fmt.Errorf("failed to determine repository root: %w", err)
+		}
+		path = filepath.Join(repoRoot, path)
+	}
+
+	return path, nil
+}
+
+func (cpr *CreatePullRequest) validateTemplateFile() error {
+	if cpr.Cfg.TemplatePath == "" {
+		return nil // No template to validate
+	}
+
+	templatePath, err := cpr.normalizeTemplatePath(cpr.Cfg.TemplatePath)
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(templatePath); err != nil {
+		if errors.Is(err, fs.ErrNotExist) {
+			return fmt.Errorf("template file does not exist: %s", cpr.Cfg.TemplatePath)
+		}
+
+		return fmt.Errorf("error accessing template file %s: %w", cpr.Cfg.TemplatePath, err)
+	}
+
+	return nil
+}
+
 // Execute executes the create pull request use case
 func (cpr CreatePullRequest) Execute() error {
+	// Validate template if specified
+	if err := cpr.validateTemplateFile(); err != nil {
```

Comment:
nitpick: with the thought above, I would pass to this function the configuration directly.
```suggestion
	if err := validateTemplateFile(cpr.Cfg.TemplatePath); err != nil {
```